react/features/status-bar/components/BaseStatusIcon.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+
+import UIUtil from '../../../../modules/UI/util/UIUtil';
```

Comment:
I think maybe that's why we thought about using inline dialog instead for tooltips?